common/web/gesture-recognizer/src/engine/headless/gestureSource.ts

```diff
@@ -369,8 +369,9 @@ export class GestureSourceSubview<
     const baseClientRect = this.recognizerConfigStack[0].targetRoot.getBoundingClientRect();
 
     return {
-      x: currentClientRect.x - baseClientRect.x,
-      y: currentClientRect.y - baseClientRect.y
+      // x, y not available in Chrome 35... but left and top are.
+      x: currentClientRect.left - baseClientRect.left,
+      y: currentClientRect.top - baseClientRect.top
```

Comment:
Are there other usages of getBoundingClientRect().x and .y are used?

What about 
https://github.com/keymanapp/keyman/blob/1c216bbb21b3e1c4634b0599b04ffaa2b53aa9fb/common/web/gesture-recognizer/src/engine/configuration/paddedZoneSource.ts#L126-L130

Reply:
Fortunately, it appears that in the other uses, I did go with `.left` and `.top` instead.